webpack.config.js

```diff
@@ -85,13 +85,6 @@ const webConfig = {
 			test: /\.ts$/,
 			exclude: /node_modules/,
 			use: [{
-                // vscode-nls-dev loader:
```

Comment:
Please put the tracking VSCode dev localization bug link. For localization eventually we will have to bring this back.

Reply:
https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/2896662